controllers/configuration_controller.go

```diff
@@ -898,9 +1020,21 @@ func (meta *TFConfigurationMeta) assembleExecutorVolumes() []v1.Volume {
 	tfBackendVolume := meta.createTFBackendVolume()
 	executorVolumes := []v1.Volume{workingVolume, inputTFConfigurationVolume, tfBackendVolume}
 	if meta.GitCredentialsSecretReference != nil {
-		gitAuthConfigVolume := meta.createGitAuthConfigVolume()
+		gitAuthConfigVolume := meta.createSecretOrConfigMapVolume(true, meta.GitCredentialsSecretReference.Name, GitAuthConfigVolumeName)
```

Comment:
This is the same. Embed a function in a table and loop it instead of execute respectively。

Reply:
fixed